extensions/ql-vscode/src/mocks/scenarios/query-failure/5-submitVariantAnalysis.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
Do you know why this is only the fifth file in the sequence? The first request seems to be retrieving another variant analysis.

Reply:
Ooops. I'll clean that up. I reckon I had something else running in my query history.

Reply:
All done.